update/genesis/import.go

```diff
@@ -150,35 +153,37 @@ func (si *stateImport) importTransactions() error {
 			break
 		}
 
-		hash, err := core.CalculateHash(si.marshalizer, si.hasher, tx)
+		var hash []byte
+		hash, err = core.CalculateHash(si.marshalizer, si.hasher, tx)
 		if err != nil {
 			break
```

Comment:
return err here :) no more need for that check after the for.

Reply:
I wanted to do a general error wrapping at the end of the function